internal/monitors/elasticsearch/metricdocs.go

```diff
@@ -0,0 +1,89 @@
+package elasticsearch
+
+// COUNTER(counter.indices.get.total): The total number of get requests since node startup
```

Comment:
Don't need these special comments at all if you have the metadata yaml file.

Reply:
Removed